### ISSUE TITLE
Good afternoon,

I am stuck with the new genomic v...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#6814]
*Help Desk query collected when no chat operators were online. Follow up required.*

Good afternoon,

I am stuck with the new genomic view. We used to be able to select an area of the genome and be able to get the corresponding sequence as a fasta file with GBrowse. But I seem to can't find this option in JBrowse and could not find the answer in the Help. Is this option still exist or should I use the Gbrowse to do that (which I've been doing but ask for extra step as it is not the default on gene description)

Thanks for your help

visitor_name

**Reported by:** Muri** (mdes****************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org/https://www.wormbase.org/tools/genome/jbrowse-simple/?tracks=Curated_Genes,Classical_alleles,High-throughput alleles,Polymorphisms,Change-of-function alleles,Change-of-function polymorphisms,Transposon insert sites,Million Mutation Project&amp;loc=IV:8171752..8192464&amp;data=data/c_elegans_PRJNA13758&amp;highlight=">https://www.wormbase.org/tools/genome/jbrowse-simple/?tracks=Curated_Genes,Classical_alleles,High-throughput alleles,Polymorphisms,Change-of-function alleles,Change-of-function polymorphisms,Transposon insert sites,Million Mutation Project&amp;loc=IV:8171752..8192464&amp;data=data/c_elegans_PRJNA13758&amp;highlight=</a>
**Browser:** Chrome 71.0.3578.98

**Comment by scottcain** [comment]
Hi Muriel,

I think there is a good chance it was you that asked this question--your
email address got lost in the chat reporting software, so I did a WormBase
person search.

Anyway, the secret in JBrowse is to turn on the "Reference sequence (DNA)"
track, and then use the track's "Save track data" option to download a
FASTA file of the sequence whose region you are currently zoomed to.

https://www.dropbox.com/s/mr1v7zvf3ur280k/Retrieving_sequence_from_JBrowse.mov?dl=0

You can also get more annotated sequence from the "Curated genes (protein
coding)" track by right-clicking on a particular transcript, selecting
"View Sequence" and playing with the options to get different bits of
sequence associated with the transcript, including upstream sequence:

https://www.dropbox.com/s/enuje3whq2ehaxq/Transcript_sequence_viewer_JBrowse.mov?dl=0

As a side note, I'm working on a button that I will place by the "hide
track labels" button that will be a "download fasta for current view"
button, but that's a release away.

Let us know if you have any questions.

Cheers,
Scott

On Thu, Jan 3, 2019 at 10:34 AM WormBase Helpdesk <notifications@github.com>
wrote:
>
> Help Desk query collected when no chat operators were online. Follow up
required.
>
> Good afternoon,
>
> I am stuck with the new genomic view. We used to be able to select an
area of the genome and be able to get the corresponding sequence as a fasta
file with GBrowse. But I seem to can't find this option in JBrowse and
could not find the answer in the Help. Is this option still exist or should
I use the Gbrowse to do that (which I've been doing but ask for extra step
as it is not the default on gene description)
>
> Thanks for your help
>
> visitor_name
>
> Reported by: Muri** (mdes****************)
> Submitted from:
https://www.wormbase.org/tools/genome/jbrowse-simple/?tracks=Curated_Genes,Classical_alleles,High-throughput
alleles,Polymorphisms,Change-of-function alleles,Change-of-function
polymorphisms,Transposon insert sites,Million Mutation
Project&loc=IV:8171752..8192464&data=data/c_elegans_PRJNA13758&highlight=
> Browser: Chrome 71.0.3578.98
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.



--
------------------------------------------------------------------------
Scott Cain, Ph. D.                                   scott at scottcain dot
net
GMOD Coordinator (http://gmod.org/)                     216-392-3087
Ontario Institute for Cancer Research

**Comment by scottcain** [comment]
I wanted to add the continued conversation with the user (since it makes jbrowse look good :)

She wrote back: 
`It was me indeed! 
Thank you so much for the answer. I will try that then!

This extra “button” will be awesome when release as it is usually what I want to do to get the promoter region and 3’UTR of the gene. ` 

And I responded:
`In that case, you will almost certainly want to  checkout the “view sequence“ thing I mentioned, since one of the options is to get set amounts of up or downstream sequence but the convenience of a simple button (when it exists) may trump that anyway :-)`

Her:
`I just tried the view sequence! it’s really nice. I like that we can choose more precisely the amount before and after the sequence and put exon in uppercase and intron in lowercase  (or vice versa) and we can keep that when copied into ApE!

That will make my life so much easier from now on!`
